src/smc-webapp/smc-react.coffee

```diff
@@ -517,4 +619,15 @@ exports.ReactDOM = require('react-dom')
 
 if DEBUG
     smc?.redux = redux  # for convenience in the browser (mainly for debugging)
+    exports._internals =
```

Comment:
incidentally, I'm also poking around in that smc-react file. right here, I added the line `smc?.react = exports`